modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -252,4 +262,235 @@ void EditorSceneFormatImporterBlend::get_import_options(const String &p_path, Li
 #undef ADD_OPTION_ENUM
 }
 
+///////////////////////////
+
+static bool _test_blender_path(const String &p_path, String *r_err = nullptr) {
+	String path = p_path;
+#ifdef WINDOWS_ENABLED
+	path = path.plus_file("blender.exe");
+#else
+	path = path.plus_file("blender");
+#endif
+	if (!FileAccess::exists(path)) {
+		if (r_err) {
+			*r_err = TTR("Path does not contain a Blender installation.");
+		}
+		return false;
+	}
+	List<String> args;
+	args.push_back("--version");
+	String pipe;
+	Error err = OS::get_singleton()->execute(path, args, &pipe);
+	if (err != OK) {
+		if (r_err) {
+			*r_err = TTR("Can't excecute.");
+		}
+		return false;
+	} else {
```

Comment:
You don't need an `else` since the previous branch returns, so you can save one indent.